service/src/test/java/org/jboss/aerogear/unifiedpush/service/ClientInstallationServiceTest.java

```diff
@@ -101,6 +96,100 @@ public void registerDevices() {
     }
 
     @Test
+    public void registerDevicesWithCategories() {
+
+        Installation device = new Installation();
+        String deviceToken = generateFakedDeviceTokenString().toUpperCase();
+        device.setDeviceToken(deviceToken);
+        final Set<Category> categories = new HashSet<Category>(Arrays.asList(new Category("football"), new Category("football")));
```

Comment:
yeah, not sure :smile: 

I think the motivation was to simulate a device that (accidentally?) registers twice (or more often) the same category, in one request. B

But yeah, since a SET, of course :) there is only one. Happy to improve the test